benchmark/poseidon2/README.md

```diff
@@ -1,6 +1,6 @@
 # Poseidon2 Hash Benchmark
 
-```
+```shell
```

Comment:
In other `benchmark/xxx/README.md`, this is absent and I think this was done on purpose.